src/components/CustomerAppeal/CustomerAppealScreen.tsx

```diff
@@ -108,25 +110,33 @@ export const CustomerAppealScreen: FunctionComponent<NavigationProps> = ({
               descriptionAlert,
             });
           } else if (typeof policyThreshold === "string") {
-            /**
-             * Policy threshold is string when its visibility is conditional to other policy that is
-             * written on the threshold.
-             *
-             * Some assumptions that are made:
-             * - Policies with dependencies on other policies should be defined after their dependents
-             * - Policies with dependencies show up depending on whether its dependent policy's
-             *   descriptionAlert is shown
-             */
-            const dependentReason = result.find(
-              (reason) =>
-                reason.descriptionAlert && reason.category === policyThreshold
-            );
-            if (dependentReason && policy.alert) {
-              result.push({
-                category: policy.category,
-                description: policy.name,
-                descriptionAlert: policy.alert.label,
-              });
+            if (
+              !(
+                selectedIdType.validation === "PASSPORT" &&
+                policy.category === "tt-token-lost-waive"
```

Comment:
do we still need to check if `typeof policyThreshold === "string"`?

Reply:
yep still need! it checks if the threshold is dependant to other category or not!